Ecosystem Partners/Partner Integration Resources/howto-migrate-heroku-addon-to-addon-engine.md

```diff
@@ -0,0 +1,176 @@
+
+---
+
+### Introduction
+
+Integrating your Partner service with the CenturyLink Add-on Engine marketplace is similar to the Heroku Add-on Partner integration approach. 
+
+There are changes to be made to the Partner manifest and provisioning API to integrate with Add-on Engine. The following sections will describe the details around these needed changes.
+
+### Changes to Add-on service manifests
+
+There are multiple small changes to how the Add-on Engine manifest is configured compared to Heroku Add-on manifests. Here is an example Add-on Engine manifest:
+
+
+#### Example Add-on Engine Manifest
+
+```
+{
+  "id": "sudosandwich",
+  "name": "Sudo me a sandwich",
+  "plans": [
+    {
+      "id": "free",
+      "name": "free",
+      "description": "Get your free version of a sandwich"
+    }
+  ],
+  "api": {
+    "config_vars": ["MYSANDWICH"],
+    "sso_salt": "STATISTICALLY.SIGNIFICANT",
+    "production": {
+      "base_url": "https://api.sudosandwich.io/",
+      "sso_url": "https://dashboard.sudosandwich.io/"
+    },
+    "test": {
+      "base_url": "https://api.staging.sudosandwich.io/",
+      "sso_url": "https://dashboard.staging.sudosandwich.io/"
+    },
+    "password": "correcthorsebatterystaple"
+  }
+}
+```
+
+#### Manifest Attributes
+
+- `id` - The add-on id. All lowercase, no spaces or punctuation. This must be a unique name for your service. Also used as username in HTTP Basic Auth requests to partner provisioning API endpoints.
+- `name` - The friendly, short description of your service. Best if kept under 80 characters.
+- `api/config_vars` - An array of attribute names to pull from provision response and expose to consumers of the service instance such as URL and credentials. We recommend these are UPPER_CASE formatted.
+- `api/sso_salt` - Single sign-on salt for connecting with service instance service instance management partner site.
+- `api/production/base_url` - The production URL of the provisioning service API endpoint.
+- `api/production/sso_url` - The production URL to send service consumers to manage service instances on partner site.
+- `api/test/base_url` - The test URL of the provisioning service API endpoint used for local development testing.
+- `api/test/sso_url` - The test URL to send service consumers to manage service instances on partner site.
+- `api/password` - The password used by Add-on Engine to authenticate against the partner provisioning API endpoints via HTTP Basic Auth.
+- `plans/id` - The unique identifier for the plan that will be be sent to the partner provision API endpoint.
+- `plans/name` - The display name of the plan that will be offered as a version of the service.
+- `plans/description` - The friendly, short description of the plan that will be offered as a version of the service.
+
+#### Differences from a Heroku Add-on Manifest
+
+- `username`, `sso_salt`, `config_vars`, `api/production/sso_url`, `api/test/sso_url`, `plans/price` and `plans/price_unit` are not valid attributes 
```

Comment:
@andrewBrunette the differences are incorrect here. In fact, I'm not sure if there are any differences now except we don't use the SSO URLs yet. They can still put them in so when we do have access we can use them.